### PR TITLE
add publish to 'next' cdn script

### PR DESCRIPTION
**Pull request by markwolff**
Refactors the old powershell script for current CDN container

### CODE REVIEW

File: AISKU/scripts/uploadFilesToAzureStorage.ps1 [hunk 1]
```diff
@@ -0,0 +1,62 @@
+$storageAccountName = "daprodcdn";
+$containerName = "next";
+
+# get js sdk directory
```
Comment (jpiyali) [inline AISKU/scripts/uploadFilesToAzureStorage.ps1]: Please test with test subscriptions first.

File: AISKU/snippet/snippet.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ var aisdk = window[aiName] || (function (aiConfig) {
     var localDocument = document, localWindow = window, scriptText = "script";
     setTimeout(function () {
         var scriptElement = localDocument.createElement(scriptText);
-        scriptElement.src = aiConfig.url || "https://az416426.vo.msecnd.net/beta/ai.1.min.js";
+        scriptElement.src = aiConfig.url || "https://az416426.vo.msecnd.net/next/ai.1.min.js";
```
Comment (hectorhdzg) [inline AISKU/snippet/snippet.js]: What is this purpose of next?

Comment (markwolff) [reply AISKU/snippet/snippet.js]: Using a different container name going forward. Old container was called `beta`, using `next` now

File: AISKU/scripts/uploadFilesToAzureStorage.ps1 [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+$jsSdkDir = Split-Path (Split-Path $MyInvocation.MyCommand.Path) -Parent;
```
Comment (jpiyali) [inline AISKU/scripts/uploadFilesToAzureStorage.ps1]: also, write wrappers for next and beta. Eventually, you want to be able to do three deployments one by one for flighting. beta, next, scripts/xxx (maybe scripts/b)

File: AISKU/snippet/snippet.js (hunk 2 above)
Comment (hectorhdzg) [reply AISKU/snippet/snippet.js]: So next will always have latest, is that correct?

Comment (markwolff) [reply AISKU/snippet/snippet.js]: I think we will be doing it like this
`scripts/b`: latest stable version
`next`: next stable version (for flighting)
`beta`: additional experimental flighting endpoint

**Pull request merged**